### PR TITLE
BMS-375: Add support for Brand API

### PR DESCRIPTION
**Pull request by benjamin-morin**
http://jira.touchtunes.com/browse/BMS-375

Only a few GET endpoints were implemented for now.

### CODE REVIEW

File: lib/brand.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
+var
```
Comment (badaldesai) [inline lib/brand.js]: Can we use `const` :)

File: lib/brand.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	events = require('events'),
+
+	co = require('co'),
+
+	request = require('./request'),
+	validation = require('./validation');
+
+const
+	DEFAULT_HOST = 'master-brandapi.scaffold-workers-main-us.m.cld.octavelive.com',
```
Comment (badaldesai) [inline lib/brand.js]: We need to point to `develop` as default

**Review by badaldesai — CHANGES_REQUESTED** [review]

**Review by badaldesai — APPROVED** [review]

**Review by Mig-OG — APPROVED** [review]
Look good to me.

File: lib/brand.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
+const events = require('events');
+const co = require('co');
+const request = require('./request');
+const validation = require('./validation');
+
+const DEFAULT_HOST = 'master-brand-api.apps.playnetwork.com';
```
Comment (badaldesai) [inline lib/brand.js:6]: Can we change it back to develop?

Comment (gcerrott) [reply lib/brand.js:6]: After review, it looks like all supported services by the SDK have the _master_ host as a default.  From a SDK client's perspective, the SDK configuration becomes inconsistent.
So, I have suggested to Benjamin that I puts back the _master_ host name for the sake of a consistent developer experience.

**Review by gcerrott — APPROVED** [review]

**Comment by benjamin-morin** [comment]
Any update on this guys?

**Pull request merged**